psi4/src/psi4/libtrans/integraltransform_moinfo.cc

```diff
@@ -296,226 +290,223 @@ IntegralTransform::process_spaces()
             }
         }
 
-        if(print_ > 5){
+        if (print_ > 5) {
             int nAOrbs = 0;
-            outfile->Printf( "Adding arrays for space %c:-\n",moSpace->label());
-            outfile->Printf( "\n\talpha orbsPI = ");
-            for(int h = 0; h < nirreps_; nAOrbs += aOrbsPI[h], ++h)
-                outfile->Printf( "%d ", aOrbsPI[h]);
-            outfile->Printf( "\n\talpha orbSym = ");
-            for(int i = 0; i < nAOrbs; ++i) outfile->Printf( "%d ", aOrbSym[i]);
-            outfile->Printf( "\n\talpha Indexing Array = ");
-            for(int i = 0; i < nAOrbs; ++i) outfile->Printf( "%d ", aIndex[i]);
-            outfile->Printf( "\n\n");
+            outfile->Printf("Adding arrays for space %c:-\n", moSpace->label());
```

Comment:
Same as above, compressing this looks really bad.

Reply:
Fixed.